packages/functional-tests/pages/login.ts

```diff
@@ -319,10 +336,16 @@ export class LoginPage extends BaseLayout {
     await this.submit();
   }
 
-  async submit() {
-    const waitForNavigation = this.page.waitForNavigation();
+  async submit(waitForNav = true) {
+    if (waitForNav) {
```

Comment:
Would you be able to use `waitForUrl` and have it accept a RegEx of possible paths? I haven't seen these `waitForNavigation` errors pop up in a while though.

Reply:
Looks like we should be able to and that it's the recommendation now:

![image](https://github.com/mozilla/fxa/assets/13018240/f6e1f6fc-9781-4ea9-9f42-26f1cfdf597d)

I could change this for the one test, but I think that might be better in a follow up since we'd still need some conditional here until we do it across the board for all tests that use this - it looks like we call `this.submit` in `login.ts` for various things like `setTotp` and `setNewPassword` etc. so we need to pass the expected URL regex in there when we call `this.submit`.